CppExport/src/dependency_analysis/DependencyComposite.h

```diff
@@ -35,18 +35,22 @@ namespace CppExport {
 class CPPEXPORT_API DependencyComposite
 {
 	public:
-		DependencyComposite(const QString& name);
+		DependencyComposite(const QString name);
```

Comment:
You should leave this as it was, with a reference, but you should keep the cange to `name_` to be a non-reference type. This will solve your bug and will make the code be like the rest of envision - QString arguments are always passed by const reference.